src/main/io/serial.h

```diff
@@ -71,7 +71,8 @@ typedef enum {
     BAUD_1000000,
     BAUD_1500000,
     BAUD_2000000,
-    BAUD_2470000
+    BAUD_2470000,
+    BAUD_COUNT
```

Comment:
I think the introduced `BAUD_COUNT` does not interfere with other code. Here are the code snippets where it checks for previous last element `BAUD_2470000`:

```
$ grep -r "BAUD_2470000" .                                                                                   tbs/crsfv3
./src/main/blackbox/blackbox_io.c:            case BAUD_2470000:
./src/main/cli/cli.c:            if (baudRateIndex < BAUD_19200 || baudRateIndex > BAUD_2470000) {
```